mix.exs

```diff
@@ -9,7 +9,6 @@ defmodule Picosat.MixProject do
       start_permanent: Mix.env() == :prod,
       deps: deps(),
       compilers: [:elixir_make | Mix.compilers()],
-      make_makefile: "Makefile",
       make_clean: ["clean"],
```

Comment:
Perhaps we can be explicit about the target here `make_targets: ["all"]`, in case that is contributing to the problem that requires building twice.

Reply:
This did not help